events/api.py

```diff
@@ -2511,13 +2511,39 @@ def _get_queryset_from_cache_many(params, param, cache_name, operator, queryset)
     return queryset
 
 
+def _param_to_keywords(param: str):
```

Comment:
Are there any tests for this? A unit test would be nice now that there's an actual small unit to test (rather than a massive single function that does ***EVERYTHING***)

Reply:
Very good idea, considering there's a little bit of logical branching. Added also `list(set())` to drop out duplicates (like a case where user tries to search both with the new keyword and the replaced keyword.

Addressed in c3f71d1d44f1b551acf3dd5491c613547bcd9141